src/main/java/tuteez/storage/JsonAdaptedPerson.java

```diff
@@ -83,55 +84,65 @@ public JsonAdaptedPerson(Person source) {
      * @throws IllegalValueException if there were any data constraints violated in the adapted person.
      */
     public Person toModelType() throws IllegalValueException {
-        final List<Tag> personTags = new ArrayList<>();
```

Comment:
Good work with pulling out the logic so that this function becomes much more concise!